test/testdrive/primary-key-optimizations.td

```diff
@@ -103,16 +103,14 @@ $ kafka-ingest format=avro topic=t1 key-format=avro key-schema=${keyschema-2keys
 > EXPLAIN SELECT DISTINCT key1, key2 FROM v2;
 "%0 =| Get v2| Project (#0, #1)"
 
-# Make sure that having a DISTINCT or a GROUP BY confers PK semantics on upstream views
-
-> CREATE VIEW distinct_view AS SELECT DISTINCT nokey FROM t1;
```

Comment:
why is this test being removed? Shouldn't DISTINCT confer PK semantics on upstream views?

If anything, it looks like what should happen to this test is that the CREATE VIEW should become a CREATE MATERIALIZED VIEW and there should be a second line
```
> EXPLAIN TYPED SELECT * FROM distinct_view
 %0 =  | Get materialize.public.distinct_view (u55)| | types = (integer?)  | | keys = ((#0))
"
```

Reply:
I had removed the `distinct_view` view, because it was unused, but I'm happy to actually use it instead of removing the view definition.

Reply:
Done in 7af16557d (although I didn't use `TYPED` to be consistent with other tests in the file -- definitely think `TYPED` would be good to use here, but I'd leave that as a future change for now if that's alright with you).

Reply:
If you want to not use typed, change the `EXPLAIN SELECT * FROM distinct_view` to `EXPLAIN SELECT DISTINCT nokey FROM distinct_view`, which should have the expected result `"%0 =| Get distinct_view"`.

Otherwise, the test is not very meaningful.

Reply:
Thanks for catching this, it must have got lost during copy/pasting :(